swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -312,9 +312,10 @@
         signInMenuItem.title = "Initializing"
         signInMenuItem.target = nil
         signOutMenuItem.isHidden = true
-      case .signedOut:
-        signInMenuItem.title = "Sign In"
+      case .signedOut(let accountId):
+        signInMenuItem.title = ((accountId == nil) ? "Enter Account ID" : "Sign In")
```

Comment:
Would it be hard to update the iOS counterpart to this as well?

Reply:
Maybe they could/should be grouped into another PR?

Reply:
The iOS was not trivial like this because of a prompt we need to get rid of. Okay, will group that into a separate PR.

Reply:
Thought so, ok sounds good!